etcdctlv3/main.go

```diff
@@ -51,6 +51,7 @@ func init() {
 
 	rootCmd.PersistentFlags().DurationVar(&globalFlags.DialTimeout, "dial-timeout", defaultDialTimeout, "dial timeout for client connections")
 
+	rootCmd.PersistentFlags().BoolVar(&globalFlags.Insecure, "insecure-transport", true, "disables transport security for client connections")
```

Comment:
we should do this when we enable client secure gRPC by default. or user cannot use etcdctlv3 out of the box.